### PR TITLE
perf: exclude big size images on generate

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context
- [x] minor changes to exclude big images on generate

### CODE REVIEW

File: pages/why-koda.vue [hunk 1]
```diff
@@ -62,7 +60,13 @@ const sections = [
   },
   {
     id: 'offlineExhibitions',
-    images: Object.entries(glob).map(([, value]) => value.default),
```
Comment (vikiival) [inline pages/why-koda.vue:65]: Isnt easier array from? And map ?

**Review by vikiival — APPROVED** [review]

**Review by Jarsen136 — COMMENTED** [review]
It would be great if there is a bundle size comparison after exclude the image. It's not that flexible when we want to change images.

File: pages/why-koda.vue (hunk 1 above)
Comment (preschian) [reply pages/why-koda.vue:65]: updated

**Comment by preschian** [comment]
> It would be great if there is a bundle size comparison after exclude the image. It's not that flexible when we want to change images.

Regarding images, it aligns with our old issue here: https://github.com/kodadot/nft-gallery/pull/6707

| before | after |
|--------|--------|
| 276K	./.nuxt/dist/server/client.manifest.mjs | 272K    ./.nuxt/dist/server/client.manifest.mjs | 

For now, we probably don't see major performance gains; it's just incremental PRs. However, based on my test here: https://github.com/kodadot/nft-gallery/pull/11260, once we can replace all SCSS-related parts and include these changes, it can reduce our build times by about 40-60 seconds

**Comment by vikiival** [comment]
Tests are failing cc @preschian

**Comment by preschian** [comment]
> Tests are failing cc @preschian

not related with this pr, same error on main branch https://github.com/kodadot/nft-gallery/actions/workflows/playwright.yml?query=branch%3Amain

afaik error since yesterday

**Review by Jarsen136 — APPROVED** [review]

**Comment by vikiival** [comment]
> > Tests are failing cc @preschian
> 
> 
> 
> not related with this pr, same error on main branch https://github.com/kodadot/nft-gallery/actions/workflows/playwright.yml?query=branch%3Amain
> 
> 
> 
> afaik error since yesterday

Can you pls fix?

**Comment by preschian** [comment]
> > > Tests are failing cc @preschian
> > 
> > 
> > not related with this pr, same error on main branch [kodadot/nft-gallery/actions/workflows/playwright.yml?query=branch%3Amain](https://github.com/kodadot/nft-gallery/actions/workflows/playwright.yml?query=branch%3Amain)
> > afaik error since yesterday
> 
> Can you pls fix?

let's try with this https://github.com/kodadot/nft-gallery/pull/11283

**Pull request merged**